pkg/providers/amifamily/ami.go

```diff
@@ -209,9 +210,6 @@ func (p *Provider) fetchAMIsFromSSM(ctx context.Context, ssmQuery string) (strin
 	}
 	ami := aws.StringValue(output.Parameter.Value)
 	p.ssmCache.SetDefault(ssmQuery, ami)
-	if p.cm.HasChanged("ssmquery-"+ssmQuery, ami) {
```

Comment:
Why are we removing the cache-check on ssm queries? Is the idea that we just need to look at the node template and see if that's changed rather than check if the ssm query that we're using has changed?

Reply:
Yeah, I didn't see a point in logging this. My thought was that we should just log the resolved AMIs and as they change, we'll log the updated ones. Logging info on the discovered AMI based on a changing SSM query seemed immaterial to me

Reply:
Yeah, also leaks some implementation details too!